streams/readable-streams/floating-point-total-queue-size.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+if (self.importScripts) {
+  self.importScripts('/resources/testharness.js');
+}
+
+promise_test(() => {
+  const { reader, controller } = setupTestStream();
+
+  controller.enqueue(2);
+  assert_equals(controller.desiredSize, 0 - 2, 'desiredSize must be -2 after enqueueing such a chunk');
+
+  controller.enqueue(Number.MAX_SAFE_INTEGER);
+  assert_equals(controller.desiredSize, 0 - Number.MAX_SAFE_INTEGER - 2,
+    'desiredSize must be calculated using floating-point arithmetic (adding a second chunk)');
```

Comment:
Pedantry: are we being insufficiently specific by saying "floating-point arithmetic"? How about "double-precision arithmetic"?